e2e_tests/tests/fixtures/no_op/single-long-train-step.yaml

```diff
@@ -1,26 +0,0 @@
-description: noop-single-long-train-step
-checkpoint_storage:
-  type: shared_fs
-  host_path: /tmp
-  storage_path: determined-integration-checkpoints
-  save_trial_best: 30
-hyperparameters:
-  global_batch_size: 32
-  metrics_progression: decreasing
-  metrics_base: 0.9
-  metrics_sigma: 0
-  training_batch_seconds: 1
-searcher:
-  metric: validation_error
-  smaller_is_better: true
-  name: single
-  max_length:
-    batches: 100
```

Comment:
somewhere along the way, this test stopped being a "single long train step" and became "a long string of short steps" anyway.  i.e. it doesn't even do as advertised anymore.